src/Lean/Meta/PPGoal.lean

```diff
@@ -161,7 +161,6 @@ def ppGoal (mvarId : MVarId) : MetaM Format := do
         else
           let fmt := addLine fmt
           match ids, type? with
-          | [], _        => pure fmt
```

Comment:
Is this a bug fix? Or, was this case unnecessary?

Reply:
It was unnecessary as just above we check `ids.isEmpty`. Actually the match on `ids` is redundant, so I removed that as well now.